bn-models/src/models/_common/measurement.ts

```diff
@@ -6,3 +6,17 @@ export interface Measurement {
   value?: number | string;
   units?: string;
 }
+
+export class MeasurementImpl implements Measurement {
```

Comment:
The way you've done this works great. 
Originally, I was thinking we just completely omit the class implementation. The only code required would be:
```typescript
export function initMeasurement(type: string, units: string): Measurement {
  return {type, units}; 
}
```
which just returns the simple javascript object (the above is shorthand for ```return {type:type, units:units}```

In the long run, this will save us from having to implement a bunch of XYZImpl classes and constructors. Since these classes are just for storing data, I'm thinking the full implementation isn't really necessary.
At the end of the day it'll be the same result, so if I'm missing something this works fine as you have it.

Reply:
good point, yeah wasn't sure whether i needed the classes or not so just removed them.